server/src/test/java/org/apache/iotdb/db/engine/compaction/recover/RewriteCrossSpaceCompactionRecoverCompatibleTest.java

```diff
@@ -0,0 +1,240 @@
+/*
```

Comment:
maybe we should test with log that use absolute path to locate files

Reply:
Resolved